templates/federated_reporting/transfer_mrproper_items.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# A script to pull items for mrproper.py script from a given hub to local
+#
+# $@ -- one or more hub hostname/IP to pull items from
+#
+
+set -e
+
+# make sure a failure in any part of a pipe sequence is a failure
+set -o pipefail
+
+source "$(dirname "$0")/config.sh"
+source "$(dirname "$0")/log.sh"
+
+# check that we have all the variables we need
+true "${CFE_FR_SSH?undefined}"
+
+true "${CFE_FR_MRPROPER_DIR?undefined}"
+true "${CFE_FR_TRANSPORTER?undefined}"
+true "${CFE_FR_TRANSPORTER_ARGS?undefined}"
+true "${CFE_FR_FEEDER_USERNAME?undefined}"
+
+if [ $# = 0 ]; then
+  log "Must provide at least one hub hostname/IP to $0, exiting!"
+  exit 1
+fi
+
+for feeder in $@; do
+  feeder_hostname=$("$CFE_FR_SSH" $CFE_FR_SSH_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder" hostname)
+  "$CFE_FR_TRANSPORTER" $CFE_FR_TRANSPORTER_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder:/$CFE_FR_MRPROPER_DIR/${feeder_hostname}.pub" "$CFE_FR_MRPROPER_DIR" &&
+  "$CFE_FR_TRANSPORTER" $CFE_FR_TRANSPORTER_ARGS "$CFE_FR_FEEDER_USERNAME@$feeder:/$CFE_FR_MRPROPER_DIR/${feeder_hostname}.cert" "$CFE_FR_MRPROPER_DIR" ||
+    log "Failed to pull mrproper items from hub $feeder"
+done
+
+
+# ensure that hubs.cert contains up-to-date information from feeders and such
+ls -t1 $CFE_FR_MRPROPER_DIR/*.cert | head -n1 | grep -q hubs.cert || cat $(ls $CFE_FR_MRPROPER_DIR/*.cert | grep -v hubs.cert) > "$CFE_FR_MRPROPER_DIR/hubs.cert"
+
+for feeder in $@; do
+  "$CFE_FR_TRANSPORTER" $CFE_FR_TRANSPORTER_ARGS "$CFE_FR_MRPROPER_DIR/hubs.cert" "$CFE_FR_FEEDER_USERNAME@$feeder:/$CFE_FR_MRPROPER_DIR/" ||
```

Comment:
`${feeder}`

Reply:
Any particular reason for this change? I understand the need for curly braces when things after the variable name might be interpreted as part of the variable name. But why here? This `$feeder` is certainly in use in `pull_dumps_from.sh` near this file. `shellcheck` doesn't complain about this issue FWIW so just want to know where you are coming from here.

Reply:
Nothing particular, it just got a bit lost in the string for me without the curly braces. Not a biggie for sure.